source/upgrade/upgrading-mattermost-server.rst

```diff
@@ -81,87 +84,81 @@ Upgrading Mattermost Server
+  
+  **Using Bleve Search**
 
-  sudo find mattermost/ mattermost/client/ -mindepth 1 -maxdepth 1 \! \( -type d \( -path mattermost/client -o -path mattermost/client/plugins -o -path mattermost/config -o -path mattermost/logs -o -path mattermost/plugins -o -path mattermost/data \) -prune \) | sort | sudo xargs rm -r
+  If using `Bleve Search <https://docs.mattermost.com/deploy/bleve-search.html>`__, and the directory exists *within* the ``mattermost`` directory, the index directory path won't be preserved with the commands below. 
+  
+  - You can either move the bleve index directory out from the ``mattermost`` directory before upgrading or, following an upgrade, you can copy the contents of the bleve index directory from the ``backup`` directory. 
+  - You can then store that directory or re-index as preferred. 
+  - The bleve indexes can be migrated without reindexing between Mattermost versions. See our `Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html#bleve-settings-experimental>`__ documentation for details on setting the bleve index directory.
       
-11. Change ownership of the new files before copying them. For example:
+8. Change ownership of the new files before copying them. For example:
 
-.. code-block:: sh
+  .. code-block:: sh
          
-  sudo chown -hR mattermost:mattermost /tmp/mattermost-upgrade/
+    sudo chown -hR mattermost:mattermost /tmp/mattermost-upgrade/
      
 .. note::
     
   - If you didn't use ``mattermost`` as the owner and group of the install directory, run ``sudo chown -hR {owner}:{group} tmp/mattermost-upgrade/``.
   - If you're uncertain what owner or group was defined, use the ``ls -l {install-path}/mattermost/bin/mattermost`` command to obtain them.
 
-12. Copy the new files to your install directory and remove the temporary files.
+9. Copy the new files to your install directory and remove the temporary files.
 
-.. code-block:: sh
+  .. code-block:: sh
 
-  sudo cp -an /tmp/mattermost-upgrade/. mattermost/
-  sudo rm -r /tmp/mattermost-upgrade/
-  sudo rm -i /tmp/mattermost*.gz
+   sudo cp -an /tmp/mattermost-upgrade/. mattermost/
+   sudo rm -r /tmp/mattermost-upgrade/
+   sudo rm -i /tmp/mattermost*.gz
 
-.. note::
-  The ``n`` (no-clobber) flag and trailing ``.`` on source are very important.
+  .. note::
+    The ``n`` (no-clobber) flag and trailing ``.`` on source are very important.
 
-13. If you want to use port 80 to serve your server, or if you have TLS set up on your Mattermost server, you *must* activate the CAP_NET_BIND_SERVICE capability to allow the new Mattermost binary to bind to low ports. For example:
+10. If you want to use port 80 to serve your server, or if you have TLS set up on your Mattermost server, you **must** activate the CAP_NET_BIND_SERVICE capability to allow the new Mattermost binary to bind to low ports. For example:
```

Comment:
Maybe add an explicit note on port 443 (instead of or in addition to TLS) as this is a common question which arises sometimes and not everyone knows the terminology/tech behind this. Also low ports isn't really specific and could be changed to *ports lower than 1024*. This leaves less open questions to the users imo.